state-chain/pallets/cf-asset-balances/src/tests.rs

```diff
@@ -231,3 +250,24 @@ pub fn do_not_refund_if_amount_is_too_low() {
 		assert_egress(0, None);
 	});
 }
+
+#[test]
+pub fn refund_validators_solana() {
+	new_test_ext().execute_with(|| {
+		const REFUND_AMOUNT: u128 = 1_000u128;
```

Comment:
This test is testing the same case as refund_validators_btc - maybe we can write a generic test that just takes a `ForeignChain` as the argument?